Ready-Set-Action_Lab/script.js

```diff
@@ -0,0 +1,22 @@
+
+const main = () => {
+    const growMeEl = document.getElementById("grow-me")
+    growMeEl.classList.add("big")
+    const shrinkMeEl = document.getElementById("shrink-me")
+    shrinkMeEl.classList.remove("big")
+    const listItems = document.querySelectorAll("li")
+    listItems.forEach(liElement => console.log(liElement.innerText))
+    const linkEl = document.getElementsByClassName("link")[0]
```

Comment:
```suggestion
    const linkEl = document.querySelector(".link")
```

`querySelector` method can be used with class names, just remember to put the "dot" in front of the class name. That way, you don't have to get a list and then access an element of the list.